torch/csrc/distributed/rpc/request_callback_impl.h

```diff
@@ -56,6 +56,12 @@ class TORCH_API RequestCallbackImpl : public RequestCallbackNoPython {
   void processRRefBackward(
       RpcCommandBase& rpc,
       const c10::intrusive_ptr<JitFuture>& responseFuture) const override;
+
+  // Helpers to run user-defined functions, operators and other computations.
+
+  c10::intrusive_ptr<JitFuture> runJitFunction(
```

Comment:
similar as the comment in the previous PR, this does not need to be a member function?

Reply:
It will be used by the subclass later in the stack

Reply:
Oh I see, we are adding a new subclass of `RequestCallbackImpl`? IIUC, it does not has a subclass today?

Reply:
Oops, no, got confused with the previous PR. This method in fact will just be used in here, so it could perhaps be a simple function, though I think it might use some other methods in turn, and also making a method is more "consistent" with `runJitOperator`. Does this make sense?

Reply:
Yep, LGTM